clients/unshaded/src/main/java/tachyon/client/lineage/options/GetLineageInfoListOptions.java

```diff
@@ -16,40 +16,39 @@
 package tachyon.client.lineage.options;
 
 import tachyon.annotation.PublicApi;
-import tachyon.client.ClientContext;
-import tachyon.conf.TachyonConf;
 
 /**
  * The method option for retrieving a list of lineage information.
  */
 @PublicApi
 public final class GetLineageInfoListOptions {
+  /** Whether or not to list the lineage info recursively, currently not supported */
+  private boolean mRecursive;
```

Comment:
I originally wanted to have some field in all the options, but looking at it now it seems like a bad idea.